core/backend/src/test/standalone/TxnManager.test.ts

```diff
@@ -254,32 +254,16 @@ describe("TxnManager", () => {
     expect(lastMod3).not.to.equal(lastMod2);
 
     assert.isTrue(txns.isUndoPossible);
-    assert.isTrue(txns.checkUndoPossible(true));
-    assert.isTrue(txns.checkUndoPossible(false));
-    assert.isTrue(txns.checkUndoPossible());
 
-    // test the ability to undo/redo from previous sessions
-    imodel.close();
-    imodel = StandaloneDb.openFile(testFileName, OpenMode.ReadWrite);
-    txns = imodel.txns;
+    // test restarting the session, which should truncate undo history
+    imodel.nativeDb.restartTxnSession();

```

Comment:
Should this new function be exposed for use by others by TxnManager?

Reply:
probably, but I think I will still mark it @internal. It's not the kind of thing an ordinary app should do.

Reply:
added. Does that explanation sound sufficient to make it @public?